src/__tests__/RelativeTime.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import {RelativeTime} from '..'
+import {render as HTMLRender} from '@testing-library/react'
+import {axe, toHaveNoViolations} from 'jest-axe'
+import {render, behavesAsComponent, checkExports} from '../utils/testing'
+expect.extend(toHaveNoViolations)
+
+describe('RelativeTime', () => {
+  behavesAsComponent({Component: RelativeTime})
+
+  checkExports('RelativeTime', {
+    default: RelativeTime
+  })
+
+  it('should have no axe violations', async () => {
+    const {container} = HTMLRender(<RelativeTime />)
```

Comment:
What would be the best example for `relative-time` to check against with axe? Would it be something like this from the examples, potentially? 

```tsx
  it('should have no axe violations', async () => {
    const {container} = HTMLRender(<RelativeTime datetime="2014-04-01T16:30:00-08:00">April 1, 2014</RelativeTime>)
    const results = await axe(container)
    expect(results).toHaveNoViolations()
  })
```